snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/core/domain/refset/SnomedReferenceSet.java

```diff
@@ -171,4 +175,9 @@ public Request<TransactionContext, Boolean> toUpdateRequest() {
 		throw new UnsupportedOperationException("Reference sets does not support update operation yet");
 	}
 
+	public boolean hasSnomedTypeTargetComponents() {
+		return String.valueOf(CONCEPT_NUMBER).equals(getMapTargetComponentType())
+				|| String.valueOf(DESCRIPTION_NUMBER).equals(getMapTargetComponentType())
+				|| String.valueOf(RELATIONSHIP_NUMBER).equals(getMapTargetComponentType());
+	}
```

Comment:
On the 7.x branch this will always return `false` because we do not store short numbers in String form in `getMapTargetComponentType()`. Some hints:

https://github.com/b2ihealthcare/snow-owl/blob/566a69748e4476c75d0ae932de42997b8e1282fc/snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/index/entry/SnomedConceptDocument.java#L373-L375

https://github.com/b2ihealthcare/snow-owl/blob/566a69748e4476c75d0ae932de42997b8e1282fc/fhir/com.b2international.snowowl.fhir.rest.tests/src/com/b2international/snowowl/fhir/tests/endpoints/conceptmap/TestMapTypeReferenceSetCreator.java#L242-L245